equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+		},
+		"errors": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Errors occurred",
+			Elem: &schema.Resource{
+				Schema: readOperationalErrorSch(),
+			},
+		},
+	}
+}
+
+func readAdditionalInfoSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"key": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Additional information key",
+		},
+		"value": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Additional information value",
+		},
+	}
+}
+
+func readFabricConnectionResourceSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"uuid": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "TBD",
+		},
+		"href": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Connection URI information",
+		},
+		"name": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Connection name. An alpha-numeric 24 characters string which can include only hyphens and underscores",
+		},
+		"description": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Customer-provided connection description",
+		},
+		"type": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Defines the connection type like VG_VC, EVPL_VC, EPL_VC, EC_VC, GW_VC, ACCESS_EPL_VC",
+		},
+		"bandwidth": {
+			Type:        schema.TypeInt,
+			Required:    true,
+			Description: "Connection bandwidth in Mbps",
+		},
+		"is_remote": {
+			Type:        schema.TypeBool,
+			Computed:    true,
+			Description: "Connection property derived from access point locations",
+		},
+		"state": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Connection overall state",
+		},
+		"operation": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Connection type-specific operational data",
+			Elem: &schema.Resource{
+				Schema: readOperationSch(),
+			},
+		},
+		"notifications": {
+			Type:        schema.TypeList,
+			Required:    true,
+			Description: "Preferences for notifications on connection configuration or status changes",
+			Elem: &schema.Resource{
+				Schema: readNotificationSch(),
+			},
+		},
+		"additional_info": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Connection additional information",
+			Elem: &schema.Resource{
+				Schema: readAdditionalInfoSch(),
+			},
+		},
+		"order": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Order related to this connection information",
+			Elem: &schema.Resource{
+				Schema: readOrderSch(),
+			},
+		},
+		"account": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Customer account information that is associated with this connection",
+			Elem: &schema.Resource{
+				Schema: readAccountSch(),
+			},
+		},
+		"change_log": {
```

Comment:
This helps to answer one of the comments I left earlier. We probably could have omitted this field or captured it as a data source.  Since you included it, and it is computed, it's fine to keep around. We'll also get this for free in the datasource. 👍